src/main/io/vtx_smartaudio.c

```diff
@@ -856,17 +859,17 @@ static void vtxSAProcess(vtxDevice_t *vtxDevice, timeUs_t currentTimeUs)
 
     if ((sa_outstanding != SA_CMD_NONE) && (nowMs - sa_lastTransmissionMs > SMARTAUDIO_CMD_TIMEOUT)) {
         // Last command timed out
-        // dprintf(("process: resending 0x%x\r\n", sa_outstanding));
+        dprintf(("process: resending 0x%x\r\n", sa_outstanding));
```

Comment:
The dprintf calls above this are wrapped around a define, but this and the next two aren't. Is this as intended?

Reply:
Yes. The definition of `dprintf` itself depends on `USE_SMARTAUDIO_DPRINTF`. The explicit `#ifdef`s before that wrap around extra code in addition to the `dprintf` that is only needed for debugging.